README.md

```diff
@@ -1,13 +1,62 @@
+The [Vue Pivot Table](https://www.syncfusion.com/vue-components/vue-pivot-table?utm_source=github&utm_medium=listing&utm_campaign=vue-pivot-table-github-samples) is a powerful control used to organize and summarize business data and display the result in a cross-table format. A quick start Vue project that shows how to get started with the Pivot Table component. This project includes a code snippet to create a new vue application and add Syncfusion Vue Pivot Table component to it. It also includes a code snippet for the demonstration of some of its key features like formatting, filtering, field list and groupingbar.
 
-Check out this online example of the Vue Pivot Table Component:
-https://ej2.syncfusion.com/vue/demos/#/material3/pivot-table/default.html 
+## Project prerequisites
 
 Make sure that you have the latest versions of NodeJS and Visual Studio Code in your machine before starting to work on this project.
 
 ### How to run this application?
 To run this application, you need to clone the `getting-started-with-the-vue-pivottable-component` repository and then open it in Visual Studio Code. Now, simply install all the necessary react packages into your current project using the `npm install` command and run your project using the `ng serve` command.
+
+
+## Features and Benefits
+
+### Pivot Chart
+
+The [Vue Pivot Chart](https://www.syncfusion.com/vue-components/vue-pivot-table?utm_source=github&utm_medium=listing&utm_campaign=vue-pivot-table-github-samples) can easily be integrated with pivot data rendered independently, and includes support for plotting more than 20 chart types. The end-user experience is greatly enhanced with a set of user-interaction features such as drill up, drill down, zoom, pan, crosshair, trackball, events, selection, and tooltip. Highly interactive field list options are available for generating reports from the relational data dynamically.
+
+### Virtual scrolling
+
+Large amounts of data can be divided and displayed in the rows and columns of the current viewport using the virtual scrolling feature. You can navigate to a desired view by scrolling the vertical and horizontal scrollbars.
+
+### Paging
+
+Large amounts of data can be divided and displayed page by page with the paging feature. The pager UI allows you to navigate to a desired page using the navigation buttons or an input text box. You can change the row and column page sizes independently at runtime. Furthermore, you can toggle the position of the row and column pagers, modify their visibility, and visualize them using a compact view mode in Vue Pivot Table.
+
+### Exporting
+
+Export the [Vue Pivot Table](https://www.syncfusion.com/vue-components/vue-pivot-table?utm_source=github&utm_medium=listing&utm_campaign=vue-pivot-table-github-samples) data to Excel, PDF, and CSV formats. You can also customize the exported document by adding header, footer, and cell properties like type, style, and position programmatically.
+
+
```

Comment:
Remove extra space here and interlink the feature too

Reply:
Done